concent_api/core/views.py

```diff
@@ -619,6 +620,9 @@ def validate_golem_message_task_to_compute(data):
 
     validate_id_value(data.compute_task_def['task_id'], 'task_id')
 
+    validate_public_key(data.provider_public_key,  'provider_public_key')
+    validate_public_key(data.requestor_public_key, 'requestor_public_key')
```

Comment:
@rwrzesien Unrelated to this pull request but this should be refactored later: I think `data` is a left-over from the early JSON implementation. We should rename it to `message` and add a type annotation.

Reply:
:+1: